appveyor.yml

```diff
@@ -7,9 +7,9 @@ cache:
 before_test:
   - "curl -sS -ostack.zip -L --insecure http://www.stackage.org/stack/windows-x86_64"
   - "7z x stack.zip stack.exe"
-  - "curl -sS -oz3.zip -L --insecure https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-win.zip"
+  - "curl -sS -oz3.zip -L --insecure https://github.com/Z3Prover/z3/releases/download/z3-4.8.6/z3-4.8.6-x64-win.zip"
```

Comment:
does this and below need to be `4.8.7` now?

Reply:
4.8.7 hasn't been released yet, so there is not yet a windows release that incorporates this fix.